package_enterprise_test.go

```diff
@@ -0,0 +1,72 @@
+// +build all enterprise
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/playscale/goconvey/convey"
+)
+
+func TestPackageEnterprise(t *testing.T) {
+	Account := os.Getenv("CONJUR_ACCOUNT")
+	Login := os.Getenv("CONJUR_AUTHN_LOGIN")
+
+	ApplianceURL_V4 := os.Getenv("CONJUR_V4_APPLIANCE_URL")
+	SSLCert_V4 := os.Getenv("CONJUR_V4_SSL_CERTIFICATE")
+	APIKey_V4 := os.Getenv("CONJUR_V4_AUTHN_API_KEY")
+
+	Path := os.Getenv("PATH")
+
+	Convey("Given a compiled summon-conjur package", t, func() {
+		Convey("Given a v4 appliance", func() {
+			Convey("Given valid configuration", func() {
```

Comment:
The top two levels of nesting here don't seem like they add much clarity to what's being tested. A compiled summon-conjur package and a v4 appliance are both absolute requirements for this set of tests.

Can we get rid of them?

Reply:
yep this was just a leftover, i'll resolve it.